frontend/src/scenes/persons/personsLogic.ts

```diff
@@ -200,6 +205,7 @@ export const personsLogic = kea<personsLogicType<Filters, PersonLogicProps, Pers
             null as PersonType | null,
```

Comment:
Confirmed this works. Is the `person` reducer shared between the loader and the reducer? I think this will also work even with `setPerson` kept inside of the loader (haven't tested so can't say for sure...)

Reply:
The issue is that if `setPerson` is left inside of the loader, then it also impacts the `personLoading` reducer, so it would immediately set the `personLoading` to false when `loadPerson` is called.

Reply:
One trick here is to use `loadPerson` as another action in the `person` reducer, which sets it to `null`. This way you don't need to call the extra `actions.setPerson(null)` action, nor allow nulls in the function.